packages/react-component-library/cypress/specs/Select/index.spec.ts

```diff
@@ -2,40 +2,40 @@ import { describe, cy, it, before } from 'local-cypress'
 
 import selectors from '../../selectors'
 
-describe('SelectE', () => {
+describe('Select', () => {
   before(() => {
-    cy.visit('/iframe.html?id=select-experimental--default&viewMode=story')
+    cy.visit('/iframe.html?id=select--default&viewMode=story')
   })
 
   describe('when the component is focused', () => {
     before(() => {
-      cy.get(selectors.selectE.outerWrapper).click()
+      cy.get(selectors.Select.outerWrapper).click()
     })
 
     it('renders four options', () => {
-      cy.get(selectors.selectE.option).should('have.length', 4)
+      cy.get(selectors.Select.option).should('have.length', 4)
     })
 
     describe('and `th` is typed', () => {
       before(() => {
-        cy.get(selectors.selectE.outerWrapper).type('Th{enter}')
+        cy.get(selectors.Select.outerWrapper).type('Th{enter}')
         cy.get('body').click()
       })
 
       it('sets the value as the item', () => {
-        cy.get(selectors.selectE.input).should(
+        cy.get(selectors.Select.input).should(
           'have.value',
-          'This is a really, really long select option label that overflows the container when selected'
+          'This is a really, really long select option label that overflows the container when Selectd'
```

Comment:
Looks like this got caught up in the changes :smile: 
```suggestion
          'This is a really, really long select option label that overflows the container when selected'
```